crates/neon/src/types_impl/mod.rs

```diff
@@ -1154,6 +1155,50 @@ impl JsFunction {
     }
 }
 
+pub(crate) unsafe fn call_local<'a, 'b, C: Context<'a>, T, AS>(
+    cx: &mut C,
+    callee: raw::Local,
+    this: Handle<'b, T>,
+    args: AS,
+) -> JsResult<'a, JsValue>
+where
+    T: Value,
+    AS: AsRef<[Handle<'b, JsValue>]>,
+{
+    let (argc, argv) = unsafe { prepare_call(cx, args.as_ref()) }?;
+    let env = cx.env();
+    let mut result: MaybeUninit<raw::Local> = MaybeUninit::zeroed();
+
+    let status = napi::call_function(
+        env.to_raw(),
+        this.to_local(),
+        callee,
+        argc as usize,
```

Comment:
Now that the C++ backend is removed, we can update `prepare_call` to return a `usize`. We are casting this to a `u32` just to cast it back to `usize`.